src/m365/spo/commands/list/list-webhook-add.spec.ts

```diff
@@ -479,7 +479,7 @@ describe(commands.LIST_WEBHOOK_ADD, () => {
   it('passes validation if the expirationDateTime is in the future but no more than six months from now', () => {
     const currentDate: Date = new Date();
     currentDate.setMonth(currentDate.getMonth() + 4);
-    const dateString: string = currentDate.getFullYear() + "-" + currentDate.getMonth() + "-01";
+    const dateString: string = currentDate.getFullYear() + "-" + ("0" + currentDate.getMonth() + 1).slice(-2) + "-01";
```

Comment:
This change might be working now, but it will break on other months. The proper way to fix it would be:

```ts
currentDate.setDate(1);
const dateString: string = currentDate.toISOString().substr(0, 10);
```

Reply:
I've fixed the bug in another issue and it's already merged to master. If you rebase with upstream, you should no longer see it 👍

Reply:
Hi @waldekmastykarz 
I fixed too just before seeing your message :)